recipes/metplus/meta.yaml

```
@@ -0,0 +1,121 @@
+{% set version = "6.0.0" %}
+{% set met_version = "12.0.2" %}
+
+package:
+  name: metplus
+  version: {{ version }}
+
+source:
+  - url: https://github.com/dtcenter/METplus/archive/refs/tags/v{{ version }}.tar.gz
+    md5: a5612dfd025055707f4a45dff028e628
+    sha256: e9358aede2fd2abecd81806227de7b165d68fdf2fc9defcbba24df229461b155
+
+  - url: https://github.com/dtcenter/MET/archive/refs/tags/v{{ met_version }}.tar.gz
+    md5: 4b9dba80639a74f1a3ecb15357785603
+    sha256: 613c74cbe5a0afa87422e81a4cd70984238c1a0eaf1a395503486331223ede45
+    folder: MET
+
+
+build:
+  skip: True  # [win]
+  skip: True  # [py!=310]
```

```suggestion
  skip: True  # [win or py!=310]
```
Our recipe analysis pipelines cannot handle repeated keys because it's invalid yaml.

---

I would have applied this change for you, but since you have opened this MR from an oranizational fork instead of a personal one, I am not granted permission.